requirements.in

```diff
@@ -16,7 +16,7 @@ django-parler
 django-reversion
 django-searchable-encrypted-fields
 django-thesaurus>=0.0.2
-graphene-django
+graphene-django>=2.10.1
```

Comment:
could have maybe just updated all the deps 🙂   not necessary, just a suggestion for future situations like this

Reply:
Ayeaye. I was deep in a hole when I did that version bumb (which actually solved all my problems). Good suggestion 🎖️